packages/owl-bot/src/bin/commands/enqueue-copy-tasks.ts

```diff
@@ -0,0 +1,149 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import admin from 'firebase-admin';
+import {
+  enqueueCopyTask,
+  getAuthenticatedOctokit,
+  getGitHubShortLivedAccessToken,
+  getFilesModifiedBySha,
+  getLastNCommits,
+  triggerEnqueueCopyJobs,
+} from '../../core';
+import {FirestoreConfigsStore} from '../../database';
+import {promisify} from 'util';
+import {readFile} from 'fs';
+import yargs = require('yargs');
+import {logger} from 'gcf-utils';
+
+const readFileAsync = promisify(readFile);
+
+interface Args {
+  'pem-path': string;
+  'app-id': number;
+  installation: number;
+  repo: string;
+  'git-path': string;
+  project: string;
+  'firestore-project': string;
+  queue: string;
+  trigger?: string;
+}
+
+export const enqueueCopyTasks: yargs.CommandModule<{}, Args> = {
+  command: 'enqueue-copy-tasks',
+  describe:
+    'look at most recent commits to repo and enqueue copy tasks for those repos',
+  builder(yargs) {
+    return yargs
+      .option('pem-path', {
+        describe: 'provide path to private key for requesting JWT',
+        type: 'string',
+        demand: true,
+      })
+      .option('app-id', {
+        describe: 'GitHub AppID',
+        type: 'number',
+        demand: true,
+      })
+      .option('installation', {
+        describe: 'installation ID for GitHub app',
+        type: 'number',
+        demand: true,
+      })
+      .option('repo', {
```

Comment:
To keep the naming consistent with other commands, could we call this the source repo?